packages/adobe-sign/__tests__/apis/adobe-sign-api.spec.ts

```diff
@@ -40,16 +40,13 @@ describe('AdobeSignAPI', () => {
             expect(result).toMatchObject(mockData);
         });
 
-        it('should throw an InternalApiError if the HTTP client cannot be created', async () => {
+        it('should throw an AdobeApiError if the HTTP client cannot be created', async () => {
```

Comment:
I would think since the secrets manager is throwing, you'd expect whatever error the secrets manager throws. I do not think we should be converting.  We also may not need an AdobeApiError. Maybe you can explain what we get from that if we treat them as a 500 in the end anyways.

Reply:
If we want to know an error is coming from AdobeSignApi, we can just add that to the error that we are throwing, or just log it like, `AdobeSignApi.getAgreement: Error while retrieving agreement. Error: ${error}`